modules/nf-core/propr/propd/templates/propd.R

```diff
@@ -82,7 +174,8 @@ opt <- list(
     cutoff_min      = NA,                   # minimun threshold to test
     cutoff_max      = NA,                   # maximun threshold to test
     cutoff_interval = NA,                   # interval between thresholds
-    ncores          = as.integer('$task.cpus')
+    ncores          = as.integer('$task.cpus'),
+    GSEA            = '$GSEAopt'
```

Comment:
we can set gsea as false in default, and then leave it parse through ext.args below in the parameter overrride section

Reply:
I have done that now